src/test/java/hello/HelloWorldTest.java

```diff
@@ -0,0 +1,39 @@
+package hello;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.bankofspring.hello.GreetingService;
+
+public class HelloWorldTest {
```

Comment:
You could also have used the Spring JUnit runner. There is not much difference in this case, however if you were loading multiple files etc it may have been worth it.